cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -322,6 +322,19 @@ def build_source_reviewer_search(by, values, first_call=True):
             }
         ],
     },
+    {
+        "name": "Nessus",
+        "macro_configurations": [
+            {
+                "macro_name": "cs_nessus",
+                "label": "Nessus Data",
+                "search_by": "sourcetype",
+                "search_values": "nessus:pro:vuln,nessus:pro:plugin,nessus_json",
```

Comment:
is this source=nessus:pro:plugin required? seems we are not using in cyences

Reply:
right, removed